dashboard/lib/middlewares/redis_session_store.rb

```diff
@@ -61,4 +89,9 @@ def delete_session(req, sid, ...)
       result
     end
   end
+
+  class RedisSessionStore < ActionDispatch::Session::RedisStore
+    include UnnecessarySessionWritePrevention
+    include DeletedSessionPreservation
```

Comment:
We need to override the existing method
```suggestion
    prepend UnnecessarySessionWritePrevention
    prepend DeletedSessionPreservation
```

Reply:
Oh, good catch! Although now I'm wondering why my manual testing was working with this as-is :thinking: